### DISCUSSION TITLE
aerospike-query-engine

### DISCUSSION THREAD

**Discussion by Aloren** [aerospike/spring-data-aerospike#531] [Team Posts]
Hi @rbotzer 
Since aerospike-query-engine is used only by spring-data-aerospike I've cleaned it up and moved required code to the spring-data-aerospike repository. https://github.com/aerospike-community/spring-data-aerospike/pull/89
I'm not sure whether deleting aerospike-query-engine from extensions repo is correct, what do you think?

**Comment by rbotzer** [comment]
I think I'll add to its README that it is obsolete now, point at Spring Data for Aerospike and link to our docs.  Then we can leave it around for a while, removing it in a few months.   Cc Joe (@arrowplum)

**Comment by Aloren** [comment]
Thanks. That would be nice!
